src/database/models/team_item.rs

```diff
@@ -176,9 +178,17 @@ impl TeamMember {
                         accepted: m.accepted,
                         user: User {
                             id: UserId(m.user_id),
-                            github_id: m.github_id,
+                            github_id: if m.public_github {
+                                m.github_id
+                            } else {
+                                None
+                            },
                             name: m.user_name,
-                            email: m.email,
+                            email: if m.public_email {
+                                m.email
+                            } else {
+                                None
+                            },
```

Comment:
IMO a user shouldn't have an option to expose their email. I feel like the feature can be easily abused (to generate spam mailing lists). I think we should keep it private except for mods

Reply:
Why not give the option? I rather enjoy responding to those that have taken the time to email me with a query on a project.